projects/igniteui-angular/src/lib/tabs/tabs.directive.ts

```diff
@@ -166,6 +172,26 @@ export abstract class IgxTabsDirective extends IgxCarouselComponentBase implemen
     protected scrollTabHeaderIntoView() {
     }
 
+    private setAttributes(igxName: string) {

```

Comment:
We could use the `this.componentName` in this method instead of the `igxName` argument.

Reply:
Done